### PR TITLE
fix(Sheet): avoid content from rendering on top of sticky title/buttons

### PR DESCRIPTION
**Pull request by marcoskolodny**
Issue: [Link](https://jira.tid.es/browse/WEB-2016)

### CODE REVIEW

File: src/__screenshot_tests__/input-fields-screenshot-test.tsx [hunk 1]
```diff
@@ -498,6 +498,13 @@ test.each`
     const field = await screen.findByLabelText('Label');
 
     await field.click({clickCount: 3});
+
```
Comment (marcoskolodny) [inline src/__screenshot_tests__/input-fields-screenshot-test.tsx:501]: Unrelated. This test seems to be unstable, I've created a task to review it: https://jira.tid.es/browse/WEB-2047

**Review by atabel — APPROVED** [review]

**Review by pladaria — APPROVED** [review]

**Review by pladaria — APPROVED** [review]
not directly related with this PR but I found that when the screen is not large enough, the text zone becomes unreadable. In this case, the fixed footer shouldn't be fixed

image

can be reproduced here: https://mistica-jp8vz0d1w-flows-projects-65bb050e.vercel.app/?path=/story/components-modals-sheet--actions

**Review by yceballost — APPROVED** [review]
🔝

**Pull request merged**